test/common/http/conn_manager_impl_test_base.h

```diff
@@ -170,6 +174,9 @@ class HttpConnectionManagerImplTest : public testing::Test, public ConnectionMan
   NiceMock<MockFilterChainFactory> filter_factory_;
   ConnectionManagerStats stats_;
   ConnectionManagerTracingStats tracing_stats_{CONN_MAN_TRACING_STATS(POOL_COUNTER(fake_stats_))};
+  Event::MockTimer* drain_begin_timer_;
+  Network::DrainDecision::DrainCloseCb drain_close_func_;
+  bool drain_close_func_set_{false};
```

Comment:
nit: do we need this? You can check if drain_close_func_ == nullptr?

Reply:
Good point. Updated.